examples/themes/netflix-theme.html

```diff
@@ -17,6 +17,8 @@
     /* Keep the buttons from overflowing */
     min-width: 420px;
     height: 720px;
+    flex-direction: column;
```

Comment:
If this needs to be added now it makes me worry that we lost something in media-container that did this already.

Reply:
May have been lost earlier... I checked out `6b916dbd788af02836bf0dbf436f3f06c75208cc` locally (only changes were build/devops-related) and did a from scratch install + dev run and here's what I'm seeing:
<img width="1739" alt="Screen Shot 2021-09-13 at 1 24 02 PM" src="https://user-images.githubusercontent.com/2389566/133136679-db83ebe5-1a2d-48b7-bf61-b2886af182ee.png">

Reply:
`git bisect` helped me figure out that this is the commit that introduced the change in behavior

https://github.com/muxinc/media-chrome/commit/cd77c4cba7fcc8c4b6bb71f55fcfe7548805b1a1

Can verify by the commit before it:

`f8b8e888f783c7e4e4f7688a744bd1051813eb3f`

<img width="1878" alt="f8b8-netflix_2021-09-14_00-27-20" src="https://user-images.githubusercontent.com/764988/133214148-c4b972a3-e82e-4ef5-b926-7dec6541dd93.png">

`cd77c4cba7fcc8c4b6bb71f55fcfe7548805b1a1`

<img width="1878" alt="cd77_2021-09-14_00-27-35" src="https://user-images.githubusercontent.com/764988/133214184-745c3979-ab14-4b23-827d-b9bbcd70d896.png">

The specific diff is actually [this tiny change](https://github.com/muxinc/media-chrome/commit/cd77c4cba7fcc8c4b6bb71f55fcfe7548805b1a1#diff-a093df03191bf45d0036001fefb7ca7217096f130642377d56e0cd90312b3e23L68-L70)

```diff
- <div id="container">
-   <slot></slot>
- </div>
+ <slot></slot>
```

The reason is because `<media-container>` is `display: flex` (or `display: inline-flex`, either one, doesn't really matter). All the child elements of media-container were previously wrapped in a `<div>`. The actual styles being applied to `#container` didn't matter, just the fact that the slot was wrapped in a div meant that there was only 1 child inside the flexbox. By removing that container then in cases where there are multiple children stacked on top caused this theme to break.

Now I'm thinking about what the default should be. We may actually want `justify-content: flex-end;` to be applied to `media-controller`  by default, that would seem more intuitive.